database_wrapper.py

```diff
@@ -420,6 +423,32 @@ def save_audio_sample_meta_data(self, formatted_data: dict) -> bool:
 
         pass
 
+    def save_course(self, course_data: dict):
+        expected_keys = {'courseId', 'dept', 'courseNum', 'units',
```

Comment:
Does `save_course` need the `courseId` ? MySQL auto-increments the id field, right?

Reply:
Yeah if you have AUTO INCREMENT set do not need to supply the `courseId`

Reply:
Sounds good, `save_course` now uses auto incrementing id and checks if `Course` already exists by instead matching `dept` and `courseNum`